### ISSUE TITLE
Kicking off!

### ISSUE THREAD

**Issue by 1point618** [wheelhouseio/curriculum-patchwork#2]
edit:

Waiting on Elizabeth for:
- [ ] breakdown of Windows versus Mac users
- [ ] hard copies of our markdown cheat sheets
- [ ] official ok for Wheelhouse to insert videos in the training modules

---

original message:

Hi @ElizabethN.

I hope your holidays went well! I had a great time out in Ohio with my girlfriend's family, although happy to be back in the City and geared up to start working on this.

I was wondering if you had time this week to connect over the phone so I could ask you some questions about the course. This is all a part of my usual course dev. process—we'll start off defining the audience for the course and what we want them to be able to do by the end of the course, and then @PeterBell and I will pair to come up with an ordered set of learning objectives for the course, and then build the content and projects from there. 

We have a good idea already about the GitHub Workflow and Intro to Git pieces, so we'll be focusing on the GitHub Pages part. 

Let me know when you're around. We can schedule ~1h. Hopefully it won't take that long, but I like giving us padding in case. 

Happy new year!
- Adrian

**Comment by ElizabethN** [comment]
> I had a great time out in Ohio with my girlfriend's family

Go Ohio! :fireworks:

> if you had time this week to connect over the phone

Yes, of course! My Thursday afternoon (1/7) is wide open. Anything work for you that day?

**Comment by 1point618** [comment]
It looks like both Peter and I are free between 1pm and 4:30pm EST. Let us know what works and we'll put it on the calendar.

**Comment by 1point618** [comment]
I actually just had something come up at 2pm, so I'm free from 1-2pm, then from 3-4:30pm. Let us know what works @ElizabethN!

**Comment by 1point618** [comment]
Hi @ElizabethN, just wanted to check in to get a time on the calendar.

In addition, Peter and I were chatting and it would be great if we could get an intro to someone on the GH-Pages team. I'd love to interview them about how they view the product, how they message it, what their common customer support issues are, etc.. A PM or someone in a product owner role would be ideal. Would you be able to make that intro?

Thanks!

**Comment by ElizabethN** [comment]
> I'm free from 1-2pm

Let's do Thursday at 1:00 pm EST then. :tada:

> it would be great if we could get an intro to someone on the GH-Pages team.

Ooh, awesome idea! I'm honestly not sure who that would be, so let me do a little poking around and get an intro for you!

**Comment by 1point618** [comment]
Hi Elizabeth! Great talking to you earlier. I wanted to send a quick recap of our discussion.
1. Audience. We decided that we're going to assume a technically literate audience for this course. Probably some knowledge of Git, GitHub, or programming, even if they can't actually _do_ it yet. This means the course won't be appropriate for the segment of Patchwork students who are very much beginners.
2. Goal of the course. By the end of the course, we want students to be able to edit the content and templates of a given Jekyll site hosted on GitHub Pages. In addition, there will be stretch goals that go beyond this for those students who complete early. Stretch goals may include being able to build a Jekyll site from scratch, being able to edit the HTML/CSS to build new designs and templates, etc..
3. Timeline. This week we're doing prep, and then I'll be focusing on building the course w/ @PeterBell over the next two weeks. After that we'll have some feedback and iteration sessions, hoping to ship the course by the end of January. I'll have a more detailed timeline for you tomorrow.
4. Other courses. For the Workflow course, we'll mostly be re-using the current Workflow course. For the command-line course, we'll be re-using some assets and re-building others. We're going to focus on getting Pages into a good place before starting on the others, because the time to build should be in terms of days, not weeks, for them.
5. There is a Patchwork happening in NYC in early Feb, which I have signed up for. Our goal is to have that be the launch of the new courses. As the time gets closer, we can coordinate my time so I can help train the teachers on how to use the courses. In addition, do we want to coordinate any sort of internal awareness campaign and/or celebration around shipping the new product? It could be a cool way to showcase to other parts of GitHub the innovations that Patchwork is spearheading.
6. Peter will reach out about access to these courses for patchwork students, how we're going to handle that from the business side of things.

Anything that I'm forgetting?

**Comment by ElizabethN** [comment]
> As the time gets closer, we can coordinate my time so I can help train the teachers on how to use the courses. In addition, do we want to coordinate any sort of internal awareness campaign and/or celebration around shipping the new product?

This may be tricky, because our hosts are different with every Patchwork. It's probably easier if you train me and then allow me to train the rest. I have an internal training session on all the pieces of the Patchwork, so this would fit in nicely with that.

And a big :+1: to shipping this with some fanfare internally. There is also talk of us blogging about it on our external blog when we ship, but we'll see how that goes.

> Anything that I'm forgetting?

We talked about gathering information about what module they might be interested in at registration. I'm thinking we could even list the requirements and prerequisites for each module and make them choose during the registration phase. That way when they get there, we can lump people accordingly, and they know what they're supposed to bring. That sound ok to you?

The things I owe you:
- breakdown of Windows versus Mac users
- hard copies of our markdown cheat sheets
- official ok for Wheelhouse to insert videos in the training modules

I that covers everything! :tada:

**Comment by PeterBell** [comment]
All sounds great @1point618 and @ElizabethN - exciting! 

In terms of other logistics, I'll ask @chowdhury to send you over an invoice later today for a 50% downpayment if that works with the balance being invoiced once we've built the courses at the end of the month? Could you let me know who the invoice should be sent to (just you? others?) and if there's anything specific you want to see on/in the invoice - otherwise I'll just put something like "Course development for Patchwork events".

**Comment by PeterBell** [comment]
In terms of student access, open to a range of options. We don't mind if everyone who takes a patchwork has access to all of the courses, but the concern is that people then tweet and share their access info so _everyone_ has access to all of the courses for free, and as we'd like to be able to sell some of the courses we'd rather avoid that. How about we start by saying that anyone at a patchwork event can do any two of the three courses? In practice we won't implement the code to limit them to two of three unless we see thousands of people signing up for all three courses, but just to give people sense that there might be a limit might be good. Any thoughts on this @ElizabethN ?

**Comment by ElizabethN** [comment]
@PeterBell @chowdhury you can send the invoice to invoices@github.com, with a cc to elizabeth@github.com. Please include the following details on your invoice:
- Amount, in USD
- Terms for payment, if there's no due date listed we will assume Net 30
- Basic details on what is being delivered here
- Banking information 
- Name and Address of party receiving payment

Our address is: GitHub, 88 Colin P Kelly Jr St., San Francisco, California, 94107. Let me know if you need any other info for the invoice.

**Comment by ElizabethN** [comment]
> How about we start by saying that anyone at a patchwork event can do any two of the three courses?

I think that works great. If we get feedback from our attendees that they would like to do more, then we can sort it out with you on what that would look like. :+1:

A point of note, we have plans to expand the number of Patchwork events in 2016, so if the number of attendees exceeds your price point, by all means let me know and we'll work something out. 

Also, I'm curious what is the cost of a typical Wheelhouse module?

**Comment by PeterBell** [comment]
Hi @ElizabethN thanks for all the info re: the invoice - I'll connect with @chowdhury and make sure that goes out today sometime!

**Comment by PeterBell** [comment]
Sounds like a plan @ElizabethN in terms of access to the modules. Will keep you posted re: the number of events, but honestly we're looking to be supportive of patchwork and drive more diversity, learning and adoption - of GitHub and Wheelhouse, so I don't see it being an issue. 

In terms of price point for a module, it really depends how much work and how complex it is, but for budgeting purposes if we thought about it as $5k/module that's probably a good ballpark. Would love to build some more once we've shipped these three if that makes any sense :)

**Comment by ElizabethN** [comment]
@peterbell oh, sorry! I was confusing. I meant what is the cost for someone to sign up for a module. :blush:

**Comment by PeterBell** [comment]
Hey @ElizabethN ohhh, sorry - makes perfect sense! Right now, we have no idea, but if I had to guess I think the sweet spot will be $5-$25 for any given module with some kind of library/unlimited access across a range of technologies eventually for say $120/yr - but we'll have to do a bunch of testing and find out!

**Comment by chowdhury** [comment]
Hey @ElizabethN just sent an invoice over to invoices@ for the first 50%; let me know if you need any additional details. Have a great weekend!

/cc @1point618 @PeterBell

**Comment by ElizabethN** [comment]
@chowdhury :tada: I'll get it approved and processed right away. Happy weekend to you, too! :dancer:

**Comment by 1point618** [comment]
> We talked about gathering information about what module they might be interested in at registration. I'm thinking we could even list the requirements and prerequisites for each module and make them choose during the registration phase. That way when they get there, we can lump people accordingly, and they know what they're supposed to bring. That sound ok to you?

That sounds great to me! I don't think we'll be able to help much with this on our side of things though, other than making the preques clear in the content.

> The things I owe you:
> 
> breakdown of Windows versus Mac users
> hard copies of our markdown cheat sheets
> official ok for Wheelhouse to insert videos in the training modules
> 
> I that covers everything! :tada:

Great! I'm going to leave this issue open until we get these then close is. Will add these to the top-level comment.

**Comment by ElizabethN** [comment]
> breakdown of Windows versus Mac users

As I mentioned, we only recently started asking about device usage, so this is not a ton of data to work with. But here's what we have so far:

![patchwork_deviceusage](https://cloud.githubusercontent.com/assets/57594/12299240/cc717446-b9e3-11e5-95fc-fea3be590efa.png)

As you can see, the distribution varies widely across countries and communities. I can share the spreadsheet with you, if you want to crunch other stats besides overall averages.

> hard copies of our markdown cheat sheets

These will be on their way soon! :blush:

> official ok for Wheelhouse to insert videos in the training modules

We are ok with using videos, as long as they are screen shots/demos. We are trying to steer clear of any faces in them, as people move on, and we want to keep the videos relevant for as long as possible. Does that make sense?

I'm going to go ahead and close this, but feel free to reopen if we need to continue the discussion on anything!

**Comment by 1point618** [comment]
That all sounds great. Pinging @PeterBell so he's aware of the videos piece since he'll be recording them.
